plugin/nginx_plus_collectd.py

```diff
@@ -727,14 +738,32 @@ def get_nginx_version(self):
         Fetch the version of nginx+.
         Note, this will only return the value, not a dict.
         '''
-        return self._send_get(self.nginx_version_url)
+        if self.api_type == "newer":
+            json_response = self._send_get(self.nginx_metadata_url)
+            if isinstance(json_response, dict):
+                return json_response.get('version', None)
+
+            LOGGER.error("Unexpected response of type: %s from %s", type(json_response), self.nginx_metadata_url)
+
+            return None
+        else:
```

Comment:
nit, no need for else since we are guaranteed to return in the if block.

Reply:
We have removed the else block from here in the latest source code.